test/backend/remote.js

```diff
@@ -549,6 +549,77 @@ module.exports = function(cfg) {
   )
 
   test(
+    'bad get',
```

Comment:
nit: I'd suggest slightly more descriptive names for these tests, e.g. 'get on unknown url'